pkg/config/process.go

```diff
@@ -13,6 +13,19 @@ import (
 const (
 	// DefaultGRPCConnectionTimeoutSecs sets the default value for timeout when connecting to the agent
 	DefaultGRPCConnectionTimeoutSecs = 60
+
+	// DefaultCheckQueueSize is the defatul max amount of checks that can be buffered in memory if the forwarder can't consume them fast enough (e.g. due to network disruption)
+	// This can be fairly high as the input should get throttled by queue bytes first.
+	// Assuming we generate ~8 checks/minute (for process/network), this should allow buffering of ~30 minutes of data assuming it fits within the queue bytes memory budget
+	DefaultCheckQueueSize = 256
```

Comment:
Consider adding a `Process` prefix to these constants, because they are in a shared namespace: `config.DefaultCheckQueueSize` sounds like it applies to all checks. (Same goes for `DefaultRTCheckQueueSize`)

Reply:
That's a really good idea! I've just updated the PR with it! Thanks for the suggestion!